R/Rules-methods.R

```diff
@@ -1481,6 +1481,61 @@ setMethod("stopTrial",
           })
 
 
+## --------------------------------------------------
+## Stopping based on precision of MTD calculated as CV(MTD)
+## --------------------------------------------------
```

Comment:
(ok for now but we will delete this as part of our cleanup - so not really necessary)

Reply:
OK. Makes sense. I did not remove for now.